sdk/endpoint/auth.go

```diff
@@ -0,0 +1,55 @@
+package endpoint
```

Comment:
Rather than calling this 'auth' which is ambigous between Authentication (authn) and authorization (authz) please name it either authorization or authz.

Reply:
Sure, thank you! Will fix